scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -660,9 +669,12 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
     }
   }
 
-  def makeTuplePatParens(bodyf: => List[Pat.Arg]): Pat = autoPos {
-    val body = inParens(if (token.is[`)`]) Nil else bodyf.map(_.require[Pat]))
-    makeTuple[Pat](body, () => Lit.Unit(), Pat.Tuple(_))
+  def makeTuplePatParens(): Pat = {
```

Comment:
I moved it inside `makeTuplePatParens`, since there was only one explicit usage.